src/Makefile.am

```diff
@@ -273,6 +273,7 @@ protoc_SOURCES = google/protobuf/compiler/main.cc
 # Tests ==============================================================
 
 protoc_inputs =                                                \
+  google/protobuf/conformance.proto                            \
```

Comment:
Can we put the conformance test related code in a separate stand-alone directory? I.e., in paralle with src. I think we will need some scripts to run the conformance test for every language and it's better to put the scripts and code in a central place.